### DISCUSSION TITLE
Questions regarding DIA-NN protein inference (Part 2)

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [vdemichev/DiaNN#786] [General]
How does DIA-NN handle razor peptides?

For example, Peptide A maps to Protein A, Peptide B maps to Proteins A and B, and Peptide C maps to Protein B. If Peptides A, B, and C are all present in the sample, how will DIA-NN infer which proteins are present? 

- According to similar a thread [Questions regarding DIA-NN protein inference #316](https://github.com/vdemichev/DiaNN/discussions/316), DIA-NN using the default parameters would output Protein A, Protein B, and ProteinA;Protein B. If I want DIA-NN to output only Protein A and Protein B, what parameter should I change?

**Comment by vdemichev** [comment]
That post was from the time another protein inference algorithm was used as default. Now you will get single-protein protein groups by default, and you can switch to the old algorithm by disabling 'Heuristic protein inference'.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for your prompt reply. I am using DIA-NN version 1.8.1, but it seems like I am still getting these Protein A;Protein B protein groups. 

I am using the default parameters (Protein Inference = Genes; Heuristic Protein Inference = On). Is there anything wrong with my parameters?
<img width="207" alt="Screenshot 2023-08-28 at 23 36 26" src="https://github.com/vdemichev/DiaNN/assets/61624696/9e7a812a-93b6-49ab-9e85-60d6948fc7c8">

**Reply by vdemichev** [reply]
I guess this would be proteins which are non-distinguishable based on detected peptides? Protein.Group column?

**Reply by ec-ho-ra-mos** [reply]
This is the Protein.Group column from the report.tsv file. Do you think these Protein A;Protein B proteins are reliably identified? Or should I exclude them in my analysis? 
<img width="215" alt="Screenshot 2023-08-28 at 23 49 29" src="https://github.com/vdemichev/DiaNN/assets/61624696/b792dc65-4e93-4d85-b9d8-d8e85bea508a">

**Reply by vdemichev** [reply]
Those that are close paralogues, makes sense they are grouped together I think. I would only discard things that are for some reason incompatible with your downstream analysis. About 'reliably identified' - this is maximum parsimony protein inference (please also see the IDPicker paper), there's no guarantee it's correct, but in most cases it produces good data.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for your prompt responses. 

I am trying to do a benchmark between different DIA analysis tools. Do you think using these protein groups is reliable for benchmarking purposes? I am inclined to only include single-protein protein groups in my analysis because as users from the previous thread mentioned other DIA analysis tools do not use the same protein inferencing strategy as this one. 

Hence, I think reporting just the single-protein protein groups is much more replicable/comparable across different DIA analysis tools.

**Reply by vdemichev** [reply]
Yes, they are reliable

**Reply by vdemichev** [reply]
'I am inclined to only include single-protein protein groups' - would not recommend this.

**Reply by vdemichev** [reply]
Spectronaut & FragPipe use comparable protein grouping.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for your responses!
